README.md

```diff
@@ -457,31 +339,65 @@ This section illustrates the protocol flow for the case of communication using f
   Host: server.example.com
 ```
 
-#### Claims parameter
+#### Claims parameter (vp_jwt)
+
+Below is a non-normative example of how the `claims` parameter can be used for requesting verified presentations signed as JWT.
 
 ```json
 {
-    "vp_token": {
-      "format": "json-ld",
-      "claims":
-      {
-        "given_name": null,
-        "family_name": null,
-        "birthdate": null
+   "userinfo":
+    {
+      "vp_jwt": {
+        "claims":
```

Comment:
this syntax is not covered by the syntax definition. I suggest to use the example fron above with the credential_types element. We can extend the query syntax once we have come to a consensus re claims vs container vs ADS vs ...

Reply:
for `vp_jwt` changed to response_type; for `vp_ldp`, it is response_type + claims parameter